charts/rancher-aks-operator/1.0.100/templates/deployment.yaml

```diff
@@ -25,3 +25,17 @@ spec:
           value: {{ .Values.httpsProxy }}
         - name: NO_PROXY
           value: {{ .Values.noProxy }}
+{{- if .Values.additionalTrustedCAs }}
+        volumeMounts:
+        - mountPath: /etc/ssl/certs/ca-additional.pem
+          name: tls-ca-additional-volume
+          subPath: ca-additional.pem
+          readOnly: true
+{{- end }}
```

Comment:
Is this being split out into separate if statements incase we add more volume mounts or could they be in the same if  check?

Reply:
Good point. For now they could be in the same check.